LLVMFloatToFixed/IndirectCallConversion.cpp

```diff
@@ -0,0 +1,98 @@
+#include "llvm/ADT/SmallPtrSet.h"
+#include "llvm/IR/Constants.h"
+#include "llvm/IR/Function.h"
+#include "llvm/IR/InstIterator.h"
+#include "llvm/IR/Instructions.h"
+#include "llvm/IR/Intrinsics.h"
+#include "llvm/IR/Module.h"
+#include "llvm/Transforms/Utils/Cloning.h"
+#include "llvm/Transforms/Utils/ValueMapper.h"
+#include "LLVMFloatToFixedPass.h"
+#include "TypeUtils.h"
+
+using namespace llvm;
+using namespace taffo;
+using namespace flttofix;
+
+using handler_function = void (FloatToFixed::*)(
+    llvm::CallInst *patchedDirectCall, llvm::Function *indirectFunction);
+
+/// Map to keep track of the handled indirect functions for the conversion.
+const std::map<const std::string, handler_function> indirectCallFunctions = {
+    {"__kmpc_fork_call", &FloatToFixed::handleKmpcFork}};
```

Comment:
Better to have this map as `static`, or it might clash with the same one defined in the other cpp file at link time (when we will move to single shared object configuration)

Reply:
Can also be moved local to the only method where it is used.

Reply:
Converted it to static and local both here and in Init to be consistent in [this commit](https://github.com/HEAPLab/TAFFO-submodule-conv/pull/4/commits/9aff5890a0cba69392374938687edbc41b0470a1)